src/locales/en-US/commands.json

```diff
@@ -110,231 +110,232 @@
-        "missingIndexParameter": "You need to give me the number of the song you want to remove from the queue",
-        "songRemoved": "{{- songName}} has been removed from the queue"
+        "missingIndexParameter": "You have to give me the number of the song you want to remove from the queue.",
+        "songRemoved": "{{- songName}} has been removed from the queue!"
       },
       "jump": {
-        "commandDescription": "Jumps to a specific song in the queue",
+        "commandDescription": "Jumps to a specific song in the queue.",
         "commandUsage": "<music number>",
-        "missingIndexParameter": "You need to give me the number of the song you want to jump to",
-        "jumpedToSong": "Jumped to {{- songName}}"
+        "missingIndexParameter": "You have to give me the number of the song you want to jump to.",
+        "jumpedToSong": "Jumped to {{- songName}}."
       }
     }
   },
   "skip": {
-    "commandDescription": "Skips the current song"
+    "commandDescription": "Skips the current song."
   },
   "stop": {
-    "commandDescription": "Stops the music playback",
+    "commandDescription": "Stops the music playback.",
     "stopped": "Playback has been stopped, leaving the voice channel."
   },
   "volume": {
-    "commandDescription": "Adjusts the volume of the music playback",
+    "commandDescription": "Adjusts the volume of the music playback.",
     "commandUsage": "<0-150>",
-    "missingVolumeParameter": "You need to give me the volume level!",
-    "invalidVolumeParameter": "You need to give me a valid number!",
+    "missingVolumeParameter": "You have to give me a volume level!",
+    "invalidVolumeParameter": "You have to give me a valid number!",
     "notPlaying": "Volume can't be adjusted while not playing anything!",
     "volumeSet": "Volume set to {{volume}}%"
   },
   "help": {
-    "commandDescription": "Shows information about commands or lists them",
+    "commandDescription": "Shows information about commands or lists them.",
     "commandUsage": "[command name]",
     "listTitle": "Available commands",
     "prefix": "Prefix",
-    "youCanUse": "You can also use {{- botMention}} as a prefix",
-    "specificInformation": "Type {{helpString}} for command specific information",
+    "youCanUse": "You can also use {{- botMention}} as a prefix.",
+    "specificInformation": "Type {{helpString}} for command specific information.",
     "aliases": "Aliases",
     "subcommands": "Subcommands",
-    "noDescriptionProvided": "No description provided",
-    "noUsageProvided": "No usage provided",
-    "commandNotFound": "I couldn't find a command with that name"
+    "noDescriptionProvided": "No description provided.",
+    "noUsageProvided": "No usage provided.",
+    "commandNotFound": "I couldn't find any command with that name."
   },
   "e621": {
-    "commandDescription": "Sends random yiff from e621",
+    "commandDescription": "Sends random yiff from e621.",
```

Comment:
This could be „Sends a random yiff from e621.”

Reply:
Fix coming in next commit.